API/src/utils/postgres.ts

```diff
@@ -65,4 +66,30 @@ export const utils = {
       return false;
     }
   },
+  // Assumes currency is dollars (temporary)
+  addTip: async (user: AuthRequestBody, amount: number) => {
+    try {
+      const idResult = await client.query(
+        `SELECT id FROM accounts
+        WHERE email = $1`,
+        [user.email]
+      );
+
+      if (idResult.status === "SELECT 0") {
+        return false;
+      }
+      let id : number = +("" + idResult.rows[0][0]);
+
+      // TODO: add time field to table 
+      const tipResult = await client.query(
+        `INSERT INTO transactions (tip_amount, driver_id)
+        values ('$${amount}', ${id})`
```

Comment:
For SQL queries this interpolation (${var}) could be a risk of sql Injection. I would stick to the format you did on line 74 because the library we are using for postgresql should prevent that.